src/lib/zod/connect_webview.ts

```diff
@@ -7,6 +7,40 @@ export const connect_webview = z.object({
   accepted_providers: z.array(z.string()).optional(),
   connected_account_id: z.string().optional(),
   created_at: z.string(),
+  custom_redirect_url: z.string().nullable(),
 })
 
 export type ConnectWebview = z.infer<typeof connect_webview>
+
+export const device_providers = z.enum([
```

Comment:
Possibly able to be imported from @seamapi/types

Reply:
Wasn't able to find such schema on @seamapi/types